cds_ils/series/serializers/custom_fields.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Series custom serializer functions."""
+
+from flask import current_app
+
+
+def format_login_required_urls(metadata):
```

Comment:
I think this is duplicated right? Maybe move this code somewhere to re-use it without duplication?
You can pass as param the URLs list of the entire metadata eventually.